compiler/arser/include/arser/arser.h

```diff
@@ -310,8 +310,8 @@ class Argument
 private:
   // The '_names' vector contains all of the options specified by the user.
   // And among them, '_long_name' and '_short_name' are selected.
```

Comment:
(Optional) How about reordering on the comment as well?
```suggestion
  // And among them, '_short_name' and '_long_name' are selected.
```